src/egon/data/datasets/electrical_neighbours.py

```diff
@@ -1267,13 +1273,391 @@ def tyndp_demand():
+        ]["p_max_pu"].values[0]
+        entry = etrago.EgonPfHvGeneratorTimeseries(
+            scn_name=scn_name, generator_id=gen_id, temp_id=1, p_max_pu=serie
+        )
+
+        session.add(entry)
+        session.commit()
+
+    return
+
+def insert_loads_sq(load_sq=None, scn_name = "status2019"):
+    """
+    Copy load timeseries data from entso-e.
+
+    Returns
+    -------
+    None.
+
+    """
+    sources = config.datasets()["electrical_neighbours"]["sources"]
+    targets = config.datasets()["electrical_neighbours"]["targets"]
+
+    ################# TEMPORAL ####################
+    load_sq = pd.read_csv("data_bundle_egon_data/load_entsoe.csv", index_col="Index")
+    ################# TEMPORAL ####################
+
+    # Delete existing data
+    db.execute_sql(
+        f"""
+        DELETE FROM {targets['load_timeseries']['schema']}.
+        {targets['load_timeseries']['table']}
+        WHERE
+        scn_name = '{scn_name}'
+        AND load_id IN (
+        SELECT load_id FROM {targets['loads']['schema']}.
+        {targets['loads']['table']}
+        WHERE
+        scn_name = '{scn_name}'
+        AND carrier = 'AC'
+        AND bus NOT IN (
+            SELECT bus_i
+            FROM  {sources['osmtgmod_bus']['schema']}.
+            {sources['osmtgmod_bus']['table']}))
+        """
+    )
+
+    db.execute_sql(
+        f"""
+        DELETE FROM {targets['loads']['schema']}.
+        {targets['loads']['table']}
+        WHERE
+        scn_name = '{scn_name}'
+        AND carrier = 'AC'
+        AND bus NOT IN (
+            SELECT bus_i
+            FROM  {sources['osmtgmod_bus']['schema']}.
+            {sources['osmtgmod_bus']['table']})
+        """
+    )
+
+    # Connect to database
+    engine = db.engine()
+    session = sessionmaker(bind=engine)()
+
+    # get the corresponding bus per foreign country
+    entsoe_to_bus = entsoe_to_bus_etrago()
+
+    # Calculate and insert demand timeseries per etrago bus_id
+    for country in load_sq.columns:
+
+        load_id = db.next_etrago_id("load")
+
+        entry = etrago.EgonPfHvLoad(
+            scn_name=scn_name,
+            load_id=int(load_id),
+            carrier="AC",
+            bus=int(entsoe_to_bus[country]),
+        )
+
+        entry_ts = etrago.EgonPfHvLoadTimeseries(
+            scn_name=scn_name,
+            load_id=int(load_id),
+            temp_id=1,
+            p_set=list(load_sq[country]),
+        )
+
+        session.add(entry)
+        session.add(entry_ts)
+        session.commit()
+
+tasks = (grid, )
+
+insert_per_scenario = set()
 
 if "eGon2035" in config.settings()["egon-data"]["--scenarios"]:
-    tasks = (grid, {tyndp_generation, tyndp_demand})
+    insert_per_scenario.update([{tyndp_generation, tyndp_demand}])
+
+if "status2019" in config.settings()["egon-data"]["--scenarios"]:
+    insert_per_scenario.update([{insert_generators_sq, insert_loads_sq}])
 
+tasks = tasks + insert_per_scenario
 
```

Comment:
@ClaraBuettner (before testing) do you think it works like this? I am insecure about the {} for the parallelization....

Reply:
I think it looks good, but those things are still a bit 'try-and-error' for me...